### PR TITLE
DHFPROD-4866: Fix placement of ellipsis for array values

### PR DESCRIPTION
**Pull request by srinathgit**
### Description
Fix placement of ellipsis for array values. 

#### Checklist: 
```diff
- Note: do not change the below
```

-  ##### Owner:

- [x] JIRA_ID included in all the commit messages
- [x] PR title is in the format JIRA_ID:Title
- [x] Rebase the branch with upstream
- [x] Squashed all commits into a single commit
- [x] Added Tests
  

- ##### Reviewer:

- [x] Reviewed Tests

### CODE REVIEW

File: marklogic-data-hub-central/ui/src/config/data.config.js [hunk 1]
```diff
@@ -108,40 +108,40 @@ const entityTypeProperties = [
...
-  "echo": { "category": "custom", "signature": "echo(input)" }, 
-  "memoryLookup": { "category": "builtin", "signature": "memoryLookup(input,inputDictionary)" }, 
-  "documentLookup": { "category": "builtin", "signature": "documentLookup(input,inputDictionaryPath)" }, 
-  "parseDate": { "category": "builtin", "signature": "parseDate(value,pattern)" }, 
-  "parseDateTime": { "category": "builtin", "signature": "parseDateTime(value,pattern)" }, 
-  "add-function": { "category": "custom", "signature": "add-function(num1,num2)" }, 
-  "unparsed-text": { "category": "xpath", "signature": "unparsed-text(xs:string)" }, 
-  "month-from-dateTime": { "category": "xpath", "signature": "month-from-dateTime(xs:dateTime?)" }, "seconds-from-dateTime": { "category": "xpath", "signature": "seconds-from-dateTime(xs:dateTime?)" }, 
-  "concat": { "category": "xpath", "signature": "concat(xs:anyAtomicType?)" } 
+const mapFunctions = {
```
Comment (rjrudin) [inline marklogic-data-hub-central/ui/src/config/data.config.js:135]: This file is only used for testing, right? I recommend adding the word "test" to it somewhere - my first though on seeing this was - why isn't the server providing this data to the UI? But I believe it's just mock data for testing purposes. The filename should imply that.

File: marklogic-data-hub-central/ui/src/config/test-data.config.js [hunk 2]
```diff
@@ -173,40 +173,40 @@ const entityTypeProperties = [
   { key: 9, name: 'gender', type: 'string' }
 ];
 
-const testJSONResponse = { 
-  properties: { 
-    propName: { output: '123EAC', sourcedFrom: 'proteinId' },
+const testJSONResponse = {
+  properties: {
+    propName: { output: 'user1@marklogic.com,user2@marklogic.com, ... (300 more)', sourcedFrom: 'proteinId' },
```
Comment (bsrikan) [inline marklogic-data-hub-central/ui/src/config/test-data.config.js:178]: can we create a separate test and use an inline jsonResponse object particular to the test. This is going to affect the sorting tests. Are all the unit tests passing with this change locally?

Comment (bsrikan) [reply marklogic-data-hub-central/ui/src/config/test-data.config.js:178]: This test can rerender with 3 different responses like the screenshots above.

Comment (srinathgit) [reply marklogic-data-hub-central/ui/src/config/test-data.config.js]: Only 2 pertain to this bug for data type array. First screenshot doesn't apply. I have added tests with data pertaining only to this bug

**Pull request merged**